DylibBinder/DBFunc.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using SwiftReflector;
+using SwiftReflector.Demangling;
+using SwiftReflector.Inventory;
+using System.Collections.Generic;
+using Dynamo.SwiftLang;
+using SwiftReflector.TypeMapping;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace DylibBinder {
+	public class DBFunc {
+		public DBFunc (TLFunction tlf, bool isMutating = false, bool isProtocol = false)
+		{
+			Name = tlf.Name.Name;
+			IsStatic = GetStaticStatus (tlf.Signature);
+			IsProperty = "False";
+			HasThrows = tlf.Signature.CanThrow.ToString ();
+			IsMutating = isMutating.ToString ();
+			OperatorKind = tlf.Operator.ToString ();
+			HasInstance = (!tlf.Signature.IsConstructor && IsStatic == "False").ToString ();
```

Comment:
Properties that sound like a bool and contain a string. Not good, is going to confuse people. Also, if you think about it, you are storing a more complicated object when you can store a smaller one, 4 bytes vs the size of the string. Is confusing and the performance is worse. Store the bools and do the bool -> string later.